diff --git a/librewolf_browser/source_files/docshell/base/nsAboutRedirector.cpp b/librewolf_browser/source_files/docshell/base/nsAboutRedirector.cpp
index fbca31c..4abb0da 100644
--- a/librewolf_browser/source_files/docshell/base/nsAboutRedirector.cpp
+++ b/librewolf_browser/source_files/docshell/base/nsAboutRedirector.cpp
@@ -93,7 +93,7 @@ static const RedirEntry kRedirMap[] = {
      nsIAboutModule::ALLOW_SCRIPT |
          nsIAboutModule::URI_SAFE_FOR_UNTRUSTED_CONTENT |
          nsIAboutModule::URI_MUST_LOAD_IN_CHILD |
-         nsIAboutModule::URI_CAN_LOAD_IN_PRIVILEGEDABOUT_PROCESS},
+         nsIAboutModule::URI_CAN_LOAD_IN_PRIVILEGED_CHILD},
     {"mozilla", "chrome://global/content/mozilla.xhtml",
      nsIAboutModule::URI_SAFE_FOR_UNTRUSTED_CONTENT},
     {"neterror", "chrome://global/content/netError.xhtml",
