diff --git a/librewolf_browser/source_files/browser/branding/librewolf/configure.sh b/librewolf_browser/source_files/browser/branding/librewolf/configure.sh
index 8e60212..c99c506 100644
--- a/librewolf_browser/source_files/browser/branding/librewolf/configure.sh
+++ b/librewolf_browser/source_files/browser/branding/librewolf/configure.sh
@@ -13,7 +13,7 @@ MOZ_DATA_REPORTING=0
 MOZ_TELEMETRY_REPORTING=0
 
 # Disables addon signing
-MOZ_REQUIRE_SIGNING=0 
+# MOZ_REQUIRE_SIGNING=0
 
 # MOZ_ADOBE_EME=0 # Disables DRM (to be discussed)
 
